app/javascript/shared/components/CustomerSatisfaction.vue

```diff
@@ -100,11 +104,23 @@ export default {
         'emoji-button',
       ];
     },
-    onSubmit() {
-      this.$emit('submit', {
-        rating: this.selectedRating,
-        feedback: this.feedback,
-      });
+    async onSubmit() {
+      this.isUpdating = true;
+      try {
+        await this.$store.dispatch('message/update', {
+          submittedValues: {
+            csat_survey_response: {
+              rating: this.selectedRating,
+              feedback_message: this.feedback,
+            },
+          },
+          messageId: this.messageId,
+        });
+      } catch (error) {
+        // Ignore error
```

Comment:
Shouldn't we be showing error with `this.showAlert`?

Reply:
@nithindavid I think we are not showing any API related error messages in the widget side